lib/matplotlib/cbook.py

```diff
@@ -1734,7 +1735,7 @@ def recursive_remove(path):
                 os.removedirs(fname)
             else:
                 os.remove(fname)
-        #os.removedirs(path)
+        # os.removedirs(path)
```

Comment:
If you really want to touch this function, I'd just alias `recursive_remove` to `shutil.rmtree` (with `onerror` set to remove files) :-)  Otherwise this is kind of pointless.